.gitignore

```diff
@@ -33,6 +33,7 @@ timit_data/
 .Python
 ide_layouts/
 build/
+_build/
```

Comment:
how this one is created, well we can keep just curious what process did it :]

Reply:
I added this one, so when moving doc from build to _build and serving it with sphinx-serve. They don't get added.